katcp/testutils.py

```diff
@@ -699,12 +701,12 @@ def test_help(self, expected_requests, full_descriptions=False,
 
         if descriptions:
             if not full_descriptions:
-                got_requests = [(name, desc.split('\n')[0])
+                got_requests = [(ensure_binary(name), ensure_binary(desc).split(b'\n')[0])
```

Comment:
We should not do this for the `got_requests`.  Those are returned by other katcp code (the message `arguments`), and must already be byte strings.  It is only the user input, i.e., `expected_requests`, that we are unsure about.

Reply:
To be more clear:  we do need the `b` prefix for the split, but not the `ensure_binary` calls.